code/datums/ruins/lavaland.dm

```diff
@@ -53,6 +53,14 @@
 	cost = 20
 	allow_duplicates = FALSE
 
+/datum/map_template/ruin/lavaland/dwarf
+	name = "Strange Dorf Portal"
+	id = "dorf-man"
+	description = "A portal emitting other-worldly energies, strange pixelated images fade in and out from inside its blocky frame."
+	suffix = "lavaland_surface_dorffort.dmm"
+	cost = 0
```

Comment:
I should agree with this as a player but
Huhhh
Anyways, doesn't the generator check if the ruin is free to make when generating? it'll NEVER spawn if you do that.